cmd/main.go

```diff
@@ -35,10 +36,11 @@ func main() {
 	}
 
 	const tlsCertRoot = "/etc/pki/tls/certs"
+	const runtimeCertRoot = "/etc/docker/certs.d"
 
 	clusterCASrc := fmt.Sprintf("%s/ca.crt", builder.SecretCertsMountPath)
 	clusterCADst := fmt.Sprintf("%s/cluster.crt", tlsCertRoot)
-	err := CopyIfExists(clusterCASrc, clusterCADst)
+	err := CopyFileIfExists(clusterCASrc, clusterCADst)
```

Comment:
looking at a cluster today, this location exists:

ls -l /var/run/secrets/kubernetes.io/serviceaccount/ca.crt 
lrwxrwxrwx. 1 root root 13 Dec 12 20:58 /var/run/secrets/kubernetes.io/serviceaccount/ca.crt -> ..data/ca.crt

Reply:
perhaps that is not the service-ca though?  i'm not sure what it is....